src/main.cpp

```diff
@@ -148,7 +235,7 @@ int main ( int argc, char** argv )
         // Common options:
 
         // Initialization file -------------------------------------------------
-        if ( GetStringArgument ( argc, argv, i, "-i", "--inifile", strArgument ) )
+        if ( cmdLine.GetStringArgument ( i, CMDLN_INIFILE, strArgument ) )
```

Comment:
cmdLine sounds a bit strange and somewhat not consistent to the style - however, I don't know what the current style would want here.

Reply:
> cmdLine sounds a bit strange and somewhat not consistent to the style - however, I don't know what the current style would want here.

Getting an argument from the Command line.. so cmdLine seams logical to me...
And consistent to what style ? Unfortunately I didn't find a consistent naming style in the code yet...